mantidproject/mantid
# Enable HB3A application to adjust detector center

Description of work.

**To test**

1. Check the newly added unit test for the modified HB3A's IDF file with movable detector center;
2. Find a recent experiment to apply the user defined detector center;
3. Try to merge data with external UB matrix and export the merged data to MD workspace;



Fixes #18045.

**RELEASE NOTES**
See docs/source/release/v3.9.0/diffraction.rst.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
